QtCollider: fix method linking

brianlheim opened the pull request:
## Purpose and Motivation

Fixes #4933.

We need to use the fully qualified "QtCollider::QcCallback", otherwise the qt metatype system
doesn't find the function signature. this seems to be a deficiency in the way qt matches function
signatures, but solving it more cleverly might be difficult and introduce other bugs, so i'm just
doing the straightforward thing for now.

another problem (maybe?) raised by this is that findText on a web view that isn't visible will
lock up the interpreter. same with findText(""). i don't know why.

## Types of changes

- Documentation
- Bug fix

## To-do list

<!-- Complete an item by checking it: [x]. Add new entries to track your progress -->

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

brianlheim commented:
FYI this has been tested by the issue reporter and found to work: https://github.com/supercollider/supercollider/issues/4933#issuecomment-629789093

dyfer commented:
Thanks @brianlheim 
The fix seems to be working.
Unfortunately, tests are hit-or-miss on my system, because of timing I assume (it worked fine when I increase wait times). I was only vaguely following conversations in the past about a) async behavior in tests b) gui timing in tests, but it seems to be an issue here. Is there any way to improve this?

dyfer commented on testsuite/classlibrary/TestWebView.sc:
```diff
@@ -0,0 +1,51 @@
+			0.1.wait;
+			cond.unhang;
+		};
+		
```
When I saved this file in scide I think this whitespace was automatically deleted... also in line 27.

brianlheim commented:
thanks for the review @dyfer -- through some experimentation i found that the 'load finished' callback always fires when the html is updated, so i updated my tests to use that instead of a direct wait. also removed the whitespace, thanks for pointing that out.

brianlheim commented:
should be ready for another review now!

dyfer commented:
Thanks @brianlheim 

This is better, however the test still times out on my system, and when it does, I get an error:
```
FAIL: a TestWebView: test_findText_found - test timeout check

ERROR: Message 'not' not understood.
RECEIVER:
   nil
ARGS:
(...)
```
I believe this is because `wasFound` is still `nil`, maybe if could be initialized to `false`?
Also, would you consider increasing the timeout?

brianlheim commented:
> maybe if could be initialized to false?

i specifically want `nil` to indicate a timeout. updated so these asserts are run conditionally

> Also, would you consider increasing the timeout?

sure, i updated it to 2 seconds

thanks @dyfer ! ready for more review.
